The rpath mangling is not needed and causes pkglint to complain.

--- Remmina-1.4.36/CMakeLists.txt.orig
+++ Remmina-1.4.36/CMakeLists.txt
@@ -595,9 +595,6 @@
 set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 set(CMAKE_BUILD_RPATH_USE_ORIGIN TRUE)
-if(NOT FREEBSD)
-  set(CMAKE_INSTALL_RPATH "\$ORIGIN/../${CMAKE_INSTALL_LIBDIR}:\$ORIGIN/..")
-endif()
 
 if(GTK3_FOUND)
   # pkg_check_modules(gio REQUIRED gio-2.0) pkg_check_modules(gio-unix REQUIRED
